src/main/java/view/impl/LoginMenu.java

```diff
@@ -1,15 +1,112 @@
 package view.impl;
 
+import model.User;
+import model.UserRole;
+import service.UserService;
 import view.Menu;
 
+import java.util.Scanner;
+
 public class LoginMenu implements Menu {
+    private UserMainMenu userMainMenu;
+    private AdminMainMenu adminMainMenu;
+
+    private UserService userService;
+    private String[] items = {"1.Login", "2.Register"};
+    private Scanner scanner;
+
     @Override
     public void show() {
-
+        showItems(items);
+        System.out.println("0. Exit");
+        scanner = new Scanner(System.in);
+        while (true) {
+            int choice = scanner.nextInt();
+            switch (choice) {
+                case 1 -> loginSubMenu();
+                case 2 -> registerSubMenu();
+                case 0 -> exit();
+            }
+        }
     }
 
     @Override
     public void exit() {
+        System.exit(0);
+    }
 
+    private boolean loginCheck(String login, String password) {
+        return userService.login(login, password).isSuccessful();
     }
+
+    private void loginSubMenu() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("input login:");
+        String login = scanner.nextLine();
+        System.out.println("input password:");
+        String password = scanner.nextLine();
+        if (loginCheck(login, password)) {
+            User user = userService.login(login, password).getValue();
```

Comment:
Please, avoid calling a method twice if you can avoid it. The method can contain heavy logic (e.g. call to the database).
You can save the result of `userService.login(login, password)` to a local variable and use it for both login checking and extracting user entity.